source/game/kart/KartCollide.cc

```diff
@@ -282,4 +394,54 @@ void KartCollide::applySomeFloorMoment(f32 down, f32 rate, CollisionGroup *hitbo
     }
 }
 
+bool KartCollide::FUN_805B6A9C(CollisionData &collisionData, Hitbox &hitbox,
```

Comment:
```suggestion
bool KartCollide::FUN_805B6A9C(CollisionData &collisionData, const Hitbox &hitbox,
```

Reply:
It's technically not `const` in-game, but the only field that could be modified PROBABLY isn't going to be used ever... So I am fine making this `const`